data/json/items/classes/gun.json

```diff
@@ -19,6 +19,14 @@
     "flags": [ "FIRE_100", "NEVER_JAMS" ]
   },
   {
+    "abstract": "launcher_base",
+    "copy-from": "gun_base",
+    "type": "GUN",
+    "name": "base launcher",
+    "skill": "launcher",
+    "valid_mod_locations": [[ "accessories", 4 ],[ "barrel", 1 ],[ "sights", 1 ],[ "mechanism", 4 ]]
```

Comment:
JSON style (in this file):
`"valid_mod_locations": [ [ "accessories", 4 ], [ "barrel", 1 ], [ "sights", 1 ], [ "mechanism", 4 ] ]`